_pages/blog.html

```diff
@@ -0,0 +1,17 @@
+---
+layout: default
+permalink: /blog
+title: Blog
+description: Stay up to date with everything we're doing!
+---
+<main>
+	<main>
+		<h1>1418 Blog</h1>
```

Comment:
Highly recommend that if you're going to do this, you take advantage of Jekyll's actual blog infrastructure, rather than just writing a page yourself: https://jekyllrb.com/docs/step-by-step/08-blogging/